app/api/admin_sales/marketer.py

```diff
@@ -0,0 +1,59 @@
+from marshmallow_jsonapi import fields
+from marshmallow_jsonapi.flask import Schema
+from flask_rest_jsonapi import ResourceList
+
+from app.api.bootstrap import api
+from app.models import db
+from app.models.order import Order, OrderTicket
+from app.models.user import User
+
+from app.api.admin_sales.utils import summary
+
+
+class AdminSalesByMarketerSchema(Schema):
+    """
+    Sales summarized by marketer
+
+    Provides
+        marketer name,
+        count of tickets and total sales for orders grouped by status
+    """
+
+    class Meta:
+        type_ = 'admin-sales-by-marketer'
+        self_view = 'v1.admin_sales_by_marketer'
+
+    id = fields.String()
+    fullname = fields.String()
+    email = fields.String()
+    sales = fields.Method('calc_sales')
+
+    @staticmethod
+    def calc_sales(obj):
+        """
+        Returns sales (dictionary with total sales and ticket count) for
+        placed, completed and pending orders
+        """
+        return summary(obj.orders)
```

Comment:
Where does the `orders` attribute come from, if there isn't an order attribute in AdminSalesByMarketerSchema?

Reply:
From the join [here](https://github.com/maxlorenz/open-event-server/blob/a748d686e6ece5b9621f8c131d88989ba5d55951/app/api/admin_sales/marketer.py#L48)